modules/adding-an-odh-instance.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='adding-an-odh-instance_{context}']
+= Adding an Open Data Hub instance 
+
+[role='_abstract']
+By adding an Open Data Hub instance to your project, you can access the URL for your Open Data Hub dashboard and share it with data science users.
+
+.Prerequisites
+* You have installed the Open Data Hub Operator.
+* You have created a new project for your instance of Open Data Hub.
+
+.Procedure
+. In the OpenShift web console, select *Operators* -> *Installed Operators*.
+. Click *Project* and select the *odh* project.
```

Comment:
Similar comment to https://github.com/opendatahub-io/opendatahub-documentation/pull/55/files#r1386919685